cross/c_icap/Makefile

```diff
@@ -1,8 +1,8 @@
 PKG_NAME = c_icap
-PKG_VERS = 0.2.2
+PKG_VERS = 0.3.5
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://downloads.sourceforge.net/project/c-icap/c-icap/0.2.x
+PKG_DIST_SITE = http://downloads.sourceforge.net/project/c-icap/c-icap/0.3.x
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
```

Comment:
i have add the line : 
REQUIRED_DSM = 4.3
UNSUPPORTED_ARCHS = bromolow cedarview x86

for dsm 4.3, bromolow cedarview x86 is not supported
for dsm 5.*, package x64 is compatible with these architecture --> no real restriction